config/nunjucks/filters.js

```diff
@@ -49,6 +49,9 @@ function formatDateRange(dateRange) {
   if (!Array.isArray(dateRange) || dateRange.length !== 2) {
     return dateRange
   }
+  if (!dateRange[1]) {
```

Comment:
Feels like this fix is something separate to adding the table? It could be a separate `fix:` commit rather than being bundled into this "feature".

That way it would be better described in [the changelog](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) too.